tests/python/test_trace2.c

```diff
@@ -6,7 +6,7 @@ struct Counters { u64 stat1; };
 BPF_TABLE("hash", struct Ptr, struct Counters, stats, 1024);
 
 int count_sched(struct pt_regs *ctx) {
-  struct Ptr key = {.ptr=ctx->bx};
+  struct Ptr key = {.ptr = ctx->PT_REGS_PARM1};
```

Comment:
Nice. However, it looks like we convert those parameters to point to the proper register in pt_regs. If so, this won't help with architectures that pass parameters on the stack like @goldshtn pointed out?